jekyll/_posts/2021-01-20-community-survey-results-2020.md

```diff
@@ -0,0 +1,221 @@
+<img src="{{site.baseurl}}/assets/news/images/survey2020/40.png" alt="What types of software are you developing using Nim?" style="width:100%"/>
+
+Nim is mostly used for writing command-line programs and libraries, followed by automation/scripts, GUI, web services and data processing.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/13.png" alt="What platforms are you targeting?" style="width:100%"/>
+
+The large majority of Nim users are targeting Linux, followed by Windows and macOS.
+Compared to 2019, we see an increase of JavaScript, Android, and iOS.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/14.png" alt="Do you use Nim at work?" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/15.png" alt="If you are not using it at work yet, do you plan to in 2021?" style="width:100%"/>
+
+About 1/3 of Nim users use it at work, either exclusively (7%) or from time to time (26%).
+
+Of people who don't use Nim at work, 1/3 can't use it because their company doesn't allow it.
+Three quarters of people who are allowed to use Nim at work plan to use it in 2021.
+
+People mention stability, better tooling, more 3rd party libraries, having to mature, and better documentation as the main fields where Nim has to improve to be more accepted in their companies.
+
+
+
+
+## Learning Nim
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/17.png" alt="What learning resources, if any, did you use to learn Nim?" style="width:100%"/>
+
+Almost all of our users have read [the official tutorials](https://nim-lang.org/docs/tut1.html), about half of them have read ["Nim by Example"](https://nim-by-example.github.io/getting_started/) and/or the ["Nim in Action"](https://book.picheta.me) book.
+[Rosetta code examples](http://rosettacode.org/wiki/Category:Nim) are also a popular choice as a learning resource, followed by ["Nim Basics"](https://narimiran.github.io/nim-basics/), ["Nim Days"](https://xmonader.github.io/nimdays/) and ["Nim Notes"](https://scripter.co/notes/nim).
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/18.png" alt="What kind of learning materials does Nim need?" style="width:100%"/>
+
+If anyone would like to create some more learning materials for Nim (which is very high on the community priority list, see below under the "Nim in 2021 and beyond" section), the most wanted types of learning materials are code examples and written tutorials.
+
+
+
+## Nim community and contributions
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/19.png" alt="Have you made code contributions to the Nim project in the past?" style="width:100%"/>
+
+Compared to 2019, we see a 20% increase of people who have already contributed to Nim!
+And there are almost three times more people who haven't contributed to Nim so far but plan to do that in the future.
```

Comment:
```suggestion
And there are almost three times as many people who haven't contributed to Nim so far but plan to do that in the future.
```

The form of "three times more" should be avoided - it's ambiguous whether "three times more than `x`" means `3 * x` or `x + 3x`.

(But of course in this case, it's obvious from the graph).